api/src/routing.js

```diff
@@ -51,8 +50,8 @@ var jsonParser = bodyParser.json({ limit: '32mb' });
 process
   .on('unhandledRejection', (reason, p) => {
     logger.error('UNHANDLED REJECTION!');
-    logger.error(reason)
-    logger.error(p)
+    logger.error(reason);
```

Comment:
Nice to have some sort of prefix here for these secondary log lines. Like `Unhandled Rejection Reason: ${reason}` or someting. In case it is empty or undefined for example.

Reply:
Done.